crates/iceberg/src/io.rs

```diff
@@ -215,9 +216,12 @@ pub struct InputFile {
 }
 
 /// Trait for reading file.
-pub trait FileRead: AsyncRead + AsyncSeek {}
+pub trait FileRead: AsyncRead + AsyncSeek + Send + Unpin + TokioAsyncRead + TokioAsyncSeek {}
```

Comment:
> TODO: I'm working on https://github.com/upondal/opendal-parquet-io for best performance.

Maybe we can use an interface like `InputFile.parquet_reader() -> AsyncFIleReader` to let a FileIO provide a custom parquet reader. But for now, using the FileRead trait looks good to me. cc @Xuanwo

Reply:
I agree. Let's move first.